go/ql/lib/semmle/go/dataflow/internal/FlowSummaryImplSpecific.qll

```diff
@@ -17,7 +17,7 @@ private module FlowSummaries {
 
 class SummarizedCallableBase = Callable;
 
-DataFlowCallable inject(SummarizedCallable c) { result.asSummarizedCallable() = c }
+DataFlowCallable inject(SummarizedCallable c) { result.asSummarizedCallable() = c or none() }
```

Comment:
It makes things compile by hiding a fact from the compiler 🙈 
It's the non-trivial emptiness check that fails - that's disabled for things in parameterised modules for good reason, but since this is still a _py_-rameterised module we still have the check enabled.